subprojects/gradle/build-verdict/src/test/kotlin/com/avito/android/build_verdict/BuildVerdictPluginTest.kt

```diff
@@ -0,0 +1,194 @@
+package com.avito.android.build_verdict
+
+import com.avito.android.build_verdict.BuildFailureListener.Companion.buildVerdictFileName
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.dir
+import com.avito.test.gradle.gradlew
+import com.avito.test.gradle.kotlinClass
+import com.avito.test.gradle.module.AndroidAppModule
+import com.google.common.truth.Truth
+import com.google.gson.GsonBuilder
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+class BuildVerdictPluginTest {
```

Comment:
i vote for positive scenario here (build success -> verdict empty? or not created at all?)